packages/cli/generators/example/index.js

```diff
@@ -57,6 +57,7 @@ const EXAMPLES = {
     'An example implmenting authentication in a LoopBack application using Passport modules.',
   ),
   'todo-jwt': g.f('A modified Todo application with JWT authentication'),
+  webpack: g.f('An example to bundle @loopback/core using webpack'),
```

Comment:
I think it needs the quote. 
```suggestion
  'webpack': g.f('An example to bundle @loopback/core using webpack'),
```

Reply:
No. `prettier` will remove `'` if the key is a valid js token.